common/source/docs/common-hitec-uavcan-servos.rst

```diff
@@ -29,15 +29,15 @@ For the purposes of this documentation, a `MD950TW-UAVCAN <https://hitecnology.c
 Current Hitec UAVCAN Servos Available
 =====================================
 
--  `MD950TW <https://hitecnology.com/actuators/md-series/md950mw-20mm-coreless-titanium-gear/product>`
--  `MD65MG <https://hitecnology.com/actuators/md-series/md65mg-12mm-stantard-composite-gear/product>`
--  `MD70MH <https://hitecnology.com/actuators/md-series/md70mh-12mm-standard-metal-gear/product>`
--  `MD85MG <https://hitecnology.com/actuators/md-series/hsi-m5085mg-industrial-grade-standard-servo/product>`
--  `MD245MW <https://hitecnology.com/actuators/md-series/md245mw-17mm-metal-gear-wmagnetic-encoder-servo-actuator/product>`
--  `MD250MW <https://hitecnology.com/actuators/md-series/md250mw-15mm-metal-gear-wmagnetic-encoder-servo-actuator/product>`
--  `MD89MW <https://hitecnology.com/actuators/md-series/d-89mw-32bit-ultra-torque-micro-servo/product>`
--  `SG33BLS/BLT <https://hitecnology.com/actuators/sg-series/sg33blt-xxxx-xxxx-servo/product>`
--  SG20GL (Coming Soon as of Aug 2020)
```

Comment:
I guess it's not actually coming in Aug 2020 though .. only 4 days left!

Reply:
Right, sorry @rmackay9 not coming in Aug, just "coming soon" as of Aug :)